src/aat/resources/features/F-200 - New Access Control Create Case/S-200_Base.td.json

```diff
@@ -0,0 +1,51 @@
+{
+  "_guid_": "S-200_Base",
+  "productName": "CCD Data Store",
+  "operationName": "create case",
+  "method": "POST",
+  "uri": "/case-types/{ctid}/cases",
+
+  "users" : {
+    "invokingUser": {
+      "username": "ccd.ac.solicitor1@gmail.com",
```

Comment:
lets hide the password with an env var, can just use [[$CCD_BEFTA_CASEWORKER_1_PWD]] for now until we create new env vars in vault and jenkins file

Reply:
yes :) 👍  👍  I will sort it